staging/src/k8s.io/apiserver/pkg/apis/audit/validation/validation.go

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 package validation
 
 import (
+	"strings"
+
+	genericvalidation "k8s.io/apimachinery/pkg/api/validation"
```

Comment:
You can't refer to the package within itself, so `validation.foo` already unambiguously refers to the imported package